substrate-node/pallets/pallet-tft-price/src/lib.rs

```diff
@@ -158,6 +162,13 @@ pub mod pallet {
             AllowedOrigin::<T>::set(Some(target));
             Ok(().into())
         }
+
+        #[pallet::weight(100_000_000 + T::DbWeight::get().writes(1))]
+        pub fn set_min_tft_price(origin: OriginFor<T>, price: u32) -> DispatchResultWithPostInfo {
```

Comment:
should add test functions:
- set min price works
- set min price fails if wrong origin
- ...

Reply:
done
also add some tests for set_allowed_origin()